src/modules/aws_vpc/mappers/route.ts

```diff
@@ -154,15 +154,17 @@ export class RouteMapper extends MapperBase<Route> {
       return out;
     },
     delete: async (es: Route[], ctx: Context) => {
-      for (const e of es) {
-        if (e.gatewayId === 'local' && ctx.memo?.db?.Route?.[this.entityId(e)]) {
-          // created by AWS, can't be deleted by the user but we need to remove it from the memo
-          delete ctx.memo.db.Route[this.entityId(e)];
-          continue;
-        } else if (e.gatewayId === 'local') continue;
-        const client = (await ctx.getAwsClient(e.region)) as AWS;
-        await this.deleteRoute(client.ec2client, e);
-      }
+      await Promise.all(
+        es.map(async e => {
+          if (e.gatewayId === 'local' && ctx.memo?.db?.Route?.[this.entityId(e)]) {
+            // created by AWS, can't be deleted by the user but we need to remove it from the memo
+            delete ctx.memo.db.Route[this.entityId(e)];
+            return;
+          } else if (e.gatewayId === 'local') return;
+          const client = (await ctx.getAwsClient(e.region)) as AWS;
+          await this.deleteRoute(client.ec2client, e);
+        }),
+      );
```

Comment:
Why is this being done in parallel? We try not to do that so we don't slam into AWS API rate limiting.

Reply:
I was keeping the implementation close to the previous one. Also, I do not expect to have a lot of deletes and most of them will try to delete the `local` one that is in all routetables and that one will never reach the AWS api. But I can serilize this to avoid issues.